Solutions/Marain.Workflow.sln

```diff
@@ -1,4 +1,4 @@
-﻿
+
```

Comment:
I can't tell from GitHub's diff format: is this deliberately removing a blank that should never have been there, or accidentally adding one that shouldn't?

Reply:
Accidentally adding one it shouldn't!